Update the Transfer Learning for NN Architectures - Multi-Modal Model part (#13)
johko/computer-vision-course: course/2 Neural Network Architectures/2.3 Multi-Modal Models/README.md

# Multimodal Models

> Multimodal is a multi-disciplinary research field where we aim to develop and create model that able to solve multiple aspects at the same time in terms but not limited to text, image, video, etc.

Reviewer: ```suggestion
Multimodal is a multi-disciplinary research field where we aim to develop and create models that are able to solve multiple aspects at the same time in terms but not limited to text, image, video, etc.
```

> Example that could be model work with Image and Text like LayoutLM, CLIP, DETR, etc.

Reviewer: ```suggestion
Example of models that work with image and text data are LayoutLM, CLIP, DETR, etc.
```
